lib/deployer.rb

```diff
@@ -21,36 +24,49 @@ def initialize(environment:, repos:, tag: nil)
   end
 
   def deploy_all
-    puts "repos to deploy: #{repos.map(&:name).join(', ')}"
+    puts "Repositories: #{repos.map(&:name).join(', ')}"
 
-    repos.each do |repo|
-      puts "\n-------------------- BEGIN #{repo.name} --------------------\n"
+    results = Parallel.map(repos, progress: 'Deploying...') do |repo|
```

Comment:
We might want to hardcode a finite number of processors or parameterize the number of processors instead of using all processors.

Reply:
Added a setting, currently set to 4 processes.